lib/vbox/machine.go

```diff
@@ -15,40 +15,28 @@ import (
 	log "github.com/Sirupsen/logrus"
 	pipeline "github.com/mattn/go-pipeline"
 	virtualbox "github.com/riobard/go-virtualbox"
-	"github.com/xshellinc/iotit/lib/constant"
+	"github.com/xshellinc/iotit/lib"
 	"github.com/xshellinc/iotit/lib/repo"
+	"github.com/xshellinc/isaax-cli/dialogs"
```

Comment:
__[gometalinter]__
* [interfacer] could not import github.com/xshellinc/isaax-cli/dialogs (cannot find package "github.com/xshellinc/isaax-cli/dialogs" in any of:
* [unconvert] could not import github.com/xshellinc/isaax-cli/dialogs (cannot find package "github.com/xshellinc/isaax-cli/dialogs" in any of:

<!-- Comment by [SideCI](https://sideci.com) -->